docker/base/Dockerfile

```diff
@@ -57,6 +57,16 @@ RUN curl -sS https://releases.nixos.org/patchelf/patchelf-0.9/patchelf-0.9.tar.b
     ./configure --prefix=/usr && \
     make install
 
+# Install OpenJDK 15 for Jazzer (Java fuzzer).
```

Comment:
Maybe link to oss-fuzz/base-builder so it's clearer where this is copied from.

Reply:
Done!